ncbi/src/main/java/org/reactome/release/Utilities.java

```diff
@@ -0,0 +1,84 @@
+package org.reactome.release;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.*;
+
+/**
+ * General Utilities class.  This class contains methods needed for general manipulation of data or common operations
+ * used by the data export process.
+ * @author jweiser
+ */
+public class Utilities {
+	/**
+	 * Takes a set of Strings and splits it into a list of the requested number of sub-sets.  If an ordered set, order
+	 * is preserved.  If the set does not split evenly into the number of set-sets requested, the remainder elements
+	 * will be added one element each to the beginning sets of the returned list.
+	 * @param set Set of Strings to split
+	 * @param numOfSubSets Number of sub-sets to return
+	 * @return List of String sub-sets
+	 */
+	public static List<Set<String>> splitSet(Set<String> set, int numOfSubSets) {
+		int subSetSize = set.size() / numOfSubSets ;
+		int numberOfExtraKeys = set.size() % numOfSubSets;
+		if (numberOfExtraKeys > 0) {
+			subSetSize += 1;
+		}
+
+		List<Set<String>> splitSets = new ArrayList<>();
+
+		Set<String> subSet = new LinkedHashSet<>();
+		int keyCount = 0;
+		for(String ncbiGeneXMLNodeString : set) {
```

Comment:
The name `ncbiGeneXMLNodeString` makes some very specific assumptions inside a rather generic-looking function, no? ;)

Reply:
Yeah, this was a holdover from when this was extracted from the NCBIGene class.  Changed to be more generic :).